app/models/user.rb

```diff
@@ -4,5 +4,9 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
         :recoverable, :rememberable, :validatable, :jwt_authenticatable, jwt_revocation_strategy: JwtDenylist
   has_many :events
+  has_many :event_participants
```

Comment:
You can remove the `has_many :events` in this model. The has many is handled by the EventParticipant model.